monthorders.cpp

```diff
@@ -1178,6 +1180,15 @@ void Game::RunUnitProduce(ARegion * r,Unit * u)
 	int output = maxproduced * ItemDefs[o->item].pOut;
 	if (ItemDefs[o->item].flags & ItemType::SKILLOUT)
 		output *= level;
+	if (ItemDefs[o->item].flags & ItemType::SKILLOUT_PROGRESSIVE) {
+		if (level == 3) {
+			output *= 2;
+		}
+		if (level == 5) {
+			output *= 3;
+		}
+	}
+		
```

Comment:
What you wanted to achieve here? To have skill level rounded up produced amount just do: (level + 1) / 2

Then results will be following:
L1 = 1 item
L2 = 1 item
L3 = 2 items
L4 = 2 items
L5 = 3 items

`output *= (level + 1) / 2`

Reply:
sounds good